bindings/python/malamute/_malamute_ctypes.py

```diff
@@ -7,7 +7,7 @@
 import os, sys
 from ctypes import *
 from ctypes.util import find_library
-import czmq
+import czmq.git
```

Comment:
It looks like zproject resolves dependencies by looking on directory name. After I did mv ../czmq.git ../czmq imports were correct.